lib/pluginOverride.js

```diff
@@ -66,7 +66,9 @@ module.exports = function override (old, fn, opts) {
     instance[kFourOhFour].arrange404(instance)
   }
 
-  for (const hook of instance[kHooks].onRegister) hook.call(this, instance, opts)
```

Comment:
mmm `this` was the parent instance, while the `instance` was the new one

it does not change too much since we inherit all from the parent

Reply:
Maybe we'll need to adjust the documentation for that. As if the `instance` contains props that the `parent` does not, they might face weird situations